ARGO/include/Utilities.h

```diff
@@ -70,7 +70,13 @@ namespace Utilities
 
 	//Flow field values
 	const int MAX_FLOW_FIELD_WEIGHT = 20;
+	const float RADIANS_TO_DEGREES = 57.2958;
```

Comment:
not sure if you added this but its showing up here, glm has a function that converts deg/rad to rad/deg

Reply:
Removed.